pkgs/by-name/li/libvisual/package.nix

```
@@ -32,27 +35,26 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     autoreconfHook
     autoconf-archive
-    pkg-config
-  ];
-  buildInputs = [
-    SDL
-    glib
+    pkgconf
+    validatePkgConfig
   ];
+  buildInputs = [ glib ] ++ lib.optional (withLvTool || withExamples) SDL_compat;
 
   configureFlags =
-    lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
-      # Remove when 0.5.x is published.
-      "--disable-lv-tool"
-    ]
+    lib.optional (!withLvTool) "--disable-lv-tool"
+    ++ lib.optional (!withExamples) "--disable-examples"
```

This would also ensure that the flags are enabled when asked for.

```suggestion
    [
      (lib.enableFeature withLvTool) "lv-tool)
      (lib.enableFeature withExamples "examples)
    ]
```